docs/security-introduction.md

```diff
@@ -0,0 +1,61 @@
+# Genestack Secure Development Practices
+
+Genestack is a complete operation and deployment ecosystem for OpenStack services that hevily utilizes cloud native application like
+Kubernetes. While developing, publishing, deploying and running OpenStack services based on Genestack we aim to ensure that our engineering teams follow
+security best practices not just for OpenStack components but also for k8s and other cloud native applications used within the Genestack ecosystem.
+
+This security primer aims to outline layered security practices for Genestack, providing actionable security recommendations at every level to mitigate
+risks by securing infrastructure, platform, applications and data at each layer of the development process.
+This primer emphasizes secure development practices that complement Genestack's architecture and operational workflows.
+
+
+## Layered Security Approach
+
+Layered security ensures comprehensive protection against evolving threats by addressing risks at multiple levels. The approach applies security measures
+to both physical infrastructure and also provides security focus to the development of the application itself. The aim is to minimize a single point
+of failure compromising the entire system. This concept aligns with the cloud native environments by catagorizing security measures across the lifecycle and
+stack of the cloud native technologies.
+
+We aim to follow guidelines from CNCF that models cloud native application into distinct phases that constitute application lifecycle. Security is
```

Comment:
This does not match the Rackspace SDLC as defined here: https://raxglobal.sharepoint.com/:w:/s/OSPCStaffandProducts2/EfvShBc7n61Kp9qvguyMhnkBmBkFBCOErV52cg_eDcCNlQ?e=nPSTWZ